common-npm-packages/docker-common-v2/registryauthenticationprovider/acrauthenticationtokenprovider.ts

```diff
@@ -45,4 +48,56 @@ export default class ACRAuthenticationTokenProvider extends AuthenticationTokenP
 
         return null;
     }
+
+    private _getACRToken(AADToken: string, retryCount: number, timeToWait: number): Q.Promise<string> {
```

Comment:
Move private methods to the bottom of the file. public methods followed by private methods